physics/module_mp_thompson.F90

```diff
@@ -1509,6 +1509,14 @@ SUBROUTINE mp_gt_driver(qv, qc, qr, qi, qs, qg, ni, nr, nc,     &
             enddo
          endif
 
+         if (merra2_aerosol_aware) then
```

Comment:
Why wouldn't you simply add a `.or.` clause to line 1494 instead?  That would do the same thing since line 1495 would be false and the 3 variables would still get updated inside the IF block for merra2.  Please consider this.  And the comment above 1494 could be changed as well.

Reply:
If add .or. to line 1494, a few more lines are needed to turn off feedbacks for nwfa1d and nifa1d (lines from 1495 to  1503), which is not needed  with merra2_aerosol_aware.